src/page/address-step.page.ts

```diff
@@ -0,0 +1,13 @@
+import { $, ElementFinder } from 'protractor';
+
+export class AddressStepPage {
+  private shipping: ElementFinder;
+
+  constructor () {
+    this.shipping = $('#center_column > form > p > button > span');
```

Comment:
let's make sure the name of these variables are related to what the selector means, take a look at [this step](https://github.com/testing-community/workshop-protractor#10-page-object-model) of the workshop the variable is named as `tShirtMenu `

the same for the rest of the varialbles